src/bundle/Resources/public/js/scripts/core/date.and.time.js

```diff
@@ -0,0 +1,75 @@
+(function (global, doc, ibexa, flatpickr) {
+    const { convertDateToTimezone, formatShortDateTime } = ibexa.helpers.timezone;
+    const userTimezone = ibexa.adminUiConfig.timezone;
+    const DEFAULT_CONFIG = {
+        enableTime: true,
+        time_24hr: true,
+        formatDate: (date) => formatShortDateTime(date, null),
+    };
+    class DateAndTime {
```

Comment:
Maybe to be consistent with css `ibexa-date-time-picker`
```suggestion
    class DateTimePicker {
```